Sources/RegexBenchmark/CLI.swift

```diff
@@ -0,0 +1,33 @@
+import ArgumentParser
+
+@main
+struct Runner: ParsableCommand {
+  @Argument(help: "Names of benchmarks to run")
+  var specificBenchmarks: [String] = []
+  
+  @Option(help: "Run only once for profiling purposes")
+  var profile = false
```

Comment:
Hmm.... Sometimes it helps profiling to run many samples since you'll see what warm behavior looks like and the hot parts become more pronounced. What's the difference between this flag and a sample count of 1?